addenda32/addenda32.rst

```diff
@@ -1222,6 +1222,26 @@ In effect, ``Tag_MPextension_use`` describes the intended use of the PLDW
           architecture extension above the base architecture as indicated by
           Tag_CPU_arch.
 
+::
+
+    Tag_PAC_extension (=50), uleb128
+       0  The user did not permit this entity to use PAC/AUT instructions
+       1  The user permitted this entity to use PAC/AUT instructions in the NOP
+          space
+       2  The user permitted this entity to use PAC/AUT instructions in the NOP
+          and in the non-NOP space
+
+::
+
+   Tag_BTI_extension (=52), uleb128
+       0  The user did not permit this entity to use BTI instructions
+
```

Comment:
very minor, but to keep with the style of the previous tags, could you remove the empty lines between the options?